docs/requirements.md

```diff
@@ -7,19 +7,135 @@
 
 ## Wireframe
 
-*TODO*
-
-## Minimum Viable Product
-
-1. The game should initialize on a grid with:
-    * A movable character
-    * Moving obstacles
-    * A score counter
-    * Number of lives display
-2. Player should be able to move frog around the grid through arrow key input
-3. Score should increment for every row achieved
-4. Hitting an obstacle should result in a death sprite onscreen, reset of frog to bottom of grid, and decrement of life counter
-5. If all lives are lost a losing message should display
-6. If all frogs make it to the top of the grid a winning message should display
-    * If frog makes it to top of grid, frog stays, and new frog is reset at bottom. Score does not reset.
-7. High score should be saved in local storage and displayed during every game
+* Please See frogger.png
+
+# Minimum Viable Product
+
+## Game initialization
+* The game should initialize on a grid with:
+    ** A Movable Frog Character
+    ** Moving Obstacles.
+    ** A Score Counter
+    ** A Highscore Counter
+    ** 3 frog sprites depicting number of lives
+* Each line in the grid should be the same height that we will call TODO:tileHeight and be 16 TODO:tileWidth wide.
+
+## Map Characteristics
+
+### 1) Frog Character Starting Location
+* The frog will start at the bottom of the screen before a road of moving vehicles in a safe strip of purple grass found in frogger-sprites.png.
+
+### 2) The Road
+* Directly above the grass lane is a 5 tileHeight tall road. The road has no sprite background but is just black. It has the height of 5 tileHeight tall and the width of the purple grass. On the 'road' is motor vehicles that can be found in frogger-sprites.png.
+
+#### Motor Vehicles
+* purple cars
+* white semi-trucks
+* white dune buggies
+* white bulldozers
+* yellow taxis
+
+### 3) Another Line Of Purple Grass
+* After the 6 lane road there should be another safe strip of purple grass.
+
+### 4) Water
+* Directly above the second safe strip of grass there is a body of 'water' that is 5 tileHeight tall and is the same width as the rest of the map. The water has no sprite background but is a dark blue. By jumping on swiftly moving logs and the backs of turtles the player can guide frogger to the safety of his 'home'.
+
+#### Water Surfaces
+* 3 Turtles wide turtle lines
+* 2 Turtles wide turtle lines
+* Short Logs
+* Medium Logs
+* Long Logs
+
+### 5) Finishing Area/'Home'
+* At the very top of the map is the finishing area/'home'. Using the green grass area with 5 empty spaces. In between each space is a lilly pad which is found in TODO:file.png
```

Comment:
Are we still doing Lily Pads or open water filled with frog sprites? @kevin-moore-729

Reply:
Good call, we aren't doing lily pads. I'll fix that.